### PR TITLE
Counting candidates is hard, lets just show the money

### PR DESCRIPTION
**Pull request by LindsayYoung**
We won't have conflicting candidate counts and we still have big numbers for context

### CODE REVIEW

File: templates/search.html [hunk 1]
```diff
@@ -128,10 +128,6 @@ <h2 class="icon-heading__title widget__title"><span class="t-short">Find electio
           <h2 class="widget__title"><span class="t-short">So far in the 2016 presidential election:</span></h2>
           <div class="widget__content">
             <div>
-              <h5 class="t-data-header">Candidates running</h5>
-              <span class="t-big-data js-count"></span>
```
Comment (msecret) [inline templates/search.html]: Is the js code that is using js-count still needed elsewhere, or should it also be removed?

Comment (LindsayYoung) [inline templates/search.html]: I am not sure. I will take a look again. Thanks Marco!

**Comment by noahmanger** [comment]
@LindsayYoung did you resolve that question?

**Comment by noahmanger** [comment]
I just removed the js references to js-count. Will merge once tests pass.

**Pull request merged**